### ISSUE TITLE
couldn't find HOME environment -- expanding `~' (ArgumentError)

### ISSUE THREAD

**Issue by lockstone** [fog/fog-core#211]
[This commit](https://github.com/fog/fog-core/commit/29cc165afd4f398a41633edc09b2b178dd85602f) really messed things up for us. Any chance we can get a reversal?

``` ruby
fog-core-1.44.2/lib/fog/core/cache.rb:94:in `expand_path': couldn't find HOME environment -- expanding `~' (ArgumentError)
        from /opt/nannybot/versions/1169/vendor/bundle/ruby/2.3.0/gems/fog-core-1.44.2/lib/fog/core/cache.rb:94:in `<class:Cache>'
        from /opt/nannybot/versions/1169/vendor/bundle/ruby/2.3.0/gems/fog-core-1.44.2/lib/fog/core/cache.rb:84:in `<module:Fog>'
        from /opt/nannybot/versions/1169/vendor/bundle/ruby/2.3.0/gems/fog-core-1.44.2/lib/fog/core/cache.rb:6:in `<top (required)>'
        from /opt/nannybot/versions/1169/vendor/bundle/ruby/2.3.0/gems/fog-core-1.44.2/lib/fog/core/model.rb:2:in `require'
        from /opt/nannybot/versions/1169/vendor/bundle/ruby/2.3.0/gems/fog-core-1.44.2/lib/fog/core/model.rb:2:in `<top (required)>'
        from /opt/nannybot/versions/1169/vendor/bundle/ruby/2.3.0/gems/fog-core-1.44.2/lib/fog/core.rb:42:in `require'
        from /opt/nannybot/versions/1169/vendor/bundle/ruby/2.3.0/gems/fog-core-1.44.2/lib/fog/core.rb:42:in `<top (required)>'
        from /opt/nannybot/versions/1169/vendor/bundle/ruby/2.3.0/gems/fog-aws-0.9.4/lib/fog/aws.rb:1:in `require'
        from /opt/nannybot/versions/1169/vendor/bundle/ruby/2.3.0/gems/fog-aws-0.9.4/lib/fog/aws.rb:1:in `<top (required)>'
```

**Comment by geemus** [comment]
@lockstone  I think it should be fixed and released in v1.44.3, sorry for the breakage!

**Comment by lockstone** [comment]
Thanks for the quick turnaround!
